core/service/transaction/register.ts

```diff
@@ -0,0 +1,91 @@
+import { ITransactionService } from 'core/service/transaction';
+import { IAssetRegister, Transaction } from 'shared/model/transaction';
+import { Account } from 'shared/model/account';
+import Response from 'shared/model/response';
+import AccountRepo from 'core/repository/account';
+import config from 'shared/util/config';
+import BUFFER from 'core/util/buffer';
+import {ITableObject} from 'core/util/common';
+
+class TransactionRegisterService implements ITransactionService<IAssetRegister> {
+
+    async create(trs: Transaction<IAssetRegister>, data: IAssetRegister ): Promise<IAssetRegister> {
+        trs.recipientAddress = null;
```

Comment:
`recipientAddress` should be optional in TransactionModel, so you dont have to make it `null`.